include/z64actor.h

```diff
@@ -315,6 +315,18 @@ typedef enum {
     /* 0xFF */ ITEM00_NONE = 0xFF
 } Item00Type;
 
+#define DROP_TABLE_NUM(i) ((i) << 4)
```

Comment:
Imo this macro name feels like it would get the table num from drop params instead of shifting a table num into place

At the very least it doesn't sound like it's constructing a bitfield value